Automattic/jetpack
# Bootstrap the Jetpack Search plugin

#### Changes proposed in this Pull Request:
The PR bootstraps the Search standalone plugin. The connection process will be handled in a followup PR.

#### Jetpack product discussion
pcNPJE-Ch-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Go to a connected site with Search subscription
* Activate Jetpack and Jetpack Search plugin if not
* Ensure Instant Search, Classic Search, Search Dashboard, Customizer, JP Search Widgets and Customberg work well 
* Add `return;` as first line of code in `projects/plugins/jetpack/jetpack.php` (this skips all of Jetpack plugin but doesn't break the connection)
* Ensure Instant Search, Classic Search, Search Dashboard, Customizer, JP Search Widgets and Customberg still work well
